estela-web/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://10.96.104.122
```

Comment:
You have to restore this file changed

Reply:
Ok, Done local API url does not appear on .env.development file.